modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/utils/JavaMetadataKeyIdModelParserUtils.java

```diff
@@ -225,7 +225,9 @@ private static Optional<MetadataKeyModelParser> getKeyIdResolverModelParser(Outp
         && outputResolverModelParser == null
         && attributesResolverModelParser == null
         && inputResolverModelParsers.isEmpty()) {
-      LOGGER.warn("A Keys Resolver is being defined without defining an Output Resolver, Input Resolver nor Attributes Resolver");
+      LOGGER
```

Comment:
Remember to add the `TODO` that was associated to this log.

Reply:
sorry forgot to add this back!